### PR TITLE
Projects

### PR DESCRIPTION
**Pull request by fede-moya**
#### What does this PR do?
Create basic CRUD for the projects model
...

#### Where should the reviewer start?
`app/controllers/projects_controller.rb`
...

#### How should this be manually tested?
Test the create, read, update  actions for projects /projects
...

#### Any background context you want to provide?

...

#### What are the relevant issues?

...

#### Screenshots (if appropriate)

...

#### Checklist:

* [ ] If you've fixed a bug or added code that should be tested, add tests!

---

#### Database changes

...

#### Deployment instructions (Specific order, ENV variables, etc...)

...

### CODE REVIEW

**Comment by mconiglio** [comment]
This also fixes #54

File: spec/controllers/projects_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+  describe 'GET #edit' do
+    subject { get :edit, params: { id: project.id } }
+
+    let(:project) { FactoryBot.create(:project) }
+
+    it 'returns a successful response' do
+      expect(response).to be_successful
+    end
+
+    it 'assign the correct project' do
```
Comment (diegokloop) [inline spec/controllers/projects_controller_spec.rb]: the 's' is missing on assigns

File: spec/controllers/projects_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+  describe 'GET #show' do
+    subject { get :show, params: { id: project.id } }
+
+    let(:project) { FactoryBot.create(:project) }
+
+    it 'returns a successful response' do
+      expect(response).to be_successful
+    end
+
+    it 'assign the correct project' do
```
Comment (diegokloop) [inline spec/controllers/projects_controller_spec.rb]: the 's' is missing on assigns

File: spec/controllers/projects_controller_spec.rb [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+      expect(response).to be_successful
+    end
+
+    it { is_expected.to render_template(:new) }
+  end
+
+  describe 'POST #create' do
+    subject { post :create, params: { project: attributes } }
+
+    let(:attributes) { FactoryBot.build(:project).attributes }
```
Comment (diegokloop) [inline spec/controllers/projects_controller_spec.rb]: correct flow should be wrapped in a context block also

File: spec/controllers/projects_controller_spec.rb [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+    end
+  end
+
+  describe 'PUT #update' do
+    subject { put :update, params: { id: project.id, project: attributes } }
+
+    let(:project) { FactoryBot.create(:project, name: 'Old Project Name', state: 'active') }
+    let(:attributes) { { name: 'New Project Name', state: 'paused' } }
+
+    it 'updates the requested project' do
```
Comment (diegokloop) [inline spec/controllers/projects_controller_spec.rb]: correct flow should also be wrapped in a context block

**Pull request merged**